project_tests/security_check

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+echo "security:check"
+bin/console security:check
```

Comment:
Lost the Symfony environment, though isn't actually important in this case - see https://github.com/elifesciences/journal/pull/853/commits/ad6a48131c9bd4fe0229aba1eff2ee306487b7e4#r164969286